sql/mob_pool_mods.sql

```diff
@@ -292,6 +292,9 @@ INSERT INTO `mob_pool_mods` VALUES (4836,62,1,1); -- Maat (BLM) NO_STANDBACK
 INSERT INTO `mob_pool_mods` VALUES (4837,62,1,1); -- Maat (RNG) NO_STANDBACK
 INSERT INTO `mob_pool_mods` VALUES (5403,62,1,1); -- Maat (NIN) NO_STANDBACK
 
+INSERT INTO `mob_pool_mods` VALUES (3784,2,-1,1); -- Stray GIL_MAX: don't drop gil
+INSERT INTO `mob_pool_mods` VALUES (2614,368,100,0); -- Memory Receptacle regain 10%
```

Comment:
Do these work form onMobInitialize? the regain I know _can_ but not sure about the gil mod. There is a preference for doing them in script if they already have a mob script because we've had some discussion on getting rid of the sql tables holding them eventually.

Reply:
yes it works. just tested.